src/bp-templates/bp-nouveau/buddypress/media/albums.php

```diff
@@ -21,7 +27,9 @@
 	</div>
 
 	<?php
-	if ( bp_is_profile_albums_support_enabled() ) {
+	if ( bp_is_group() &&  bp_is_group_albums_support_enabled() ) {
```

Comment:
@mahedihasannoman Here we can use single if statement with **OR**. As a content statement for if and else is same.

Reply:
Done